packages/calcite-components/src/components/list/list.e2e.ts

```diff
@@ -339,4 +339,87 @@ describe("calcite-list", () => {
     expect(await listItemOne.getProperty("selected")).toBe(false);
     expect(await list.getProperty("selectedItems")).toHaveLength(0);
   });
+
+  describe.only("keyboard navigation", () => {
+    async function evaluateActiveElement(page: E2EPage, activeElementSelector: string): Promise<void> {
+      expect(await page.evaluate((selector) => document.activeElement?.matches(selector), activeElementSelector)).toBe(
+        true
+      );
+    }
+
+    it("should navigate via ArrowUp, ArrowDown, Home, and End", async () => {
+      const page = await newE2EPage({
+        html: html`
+          <calcite-list>
+            <calcite-list-item id="one" value="one" label="One" description="hello world"></calcite-list-item>
+            <calcite-list-item id="two" value="two" label="Two" description="hello world"></calcite-list-item>
+            <calcite-list-item
+              disabled
+              id="three"
+              value="three"
+              label="three"
+              description="hello world"
+            ></calcite-list-item>
+            <calcite-list-item
+              closable
+              closed
+              id="four"
+              value="four"
+              label="four"
+              description="hello world"
+            ></calcite-list-item>
+          </calcite-list>
+        `
+      });
+      await page.waitForChanges();
+      const list = await page.find("calcite-list");
+      await list.callMethod("setFocus");
+      await page.waitForChanges();
+      await page.waitForTimeout(0);
```

Comment:
Will the extra timeout still be needed after the focus-ensuring util is applied?

Reply:
Nope removed.